packages/angular_devkit/build_angular/test/browser/svg_spec_large.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { runTargetSpec } from '@angular-devkit/architect/testing';
+import { join, normalize, virtualFs } from '@angular-devkit/core';
+import { tap } from 'rxjs/operators';
+import { browserTargetSpec, host, outputPath } from '../utils';
+
+describe('Browser Builder allow svg', () => {
+
+  beforeEach(done => host.initialize().toPromise().then(done, done.fail));
+  afterEach(done => host.restore().toPromise().then(done, done.fail));
+
+  it('can use .component.svg as templates (uses raw-loader and does not copy .svg to dist)',
+    (done) => {
+
+    host.writeMultipleFiles({
+      './src/app.component.svg': `<svg></svg>`,
+      './src/main.ts': `declare var require: Function;
+      const x = require('./app.component.svg');
+      console.log(x);`,
+    });
+
+    host.replaceInFile(
+      'tsconfig.json',
+      '"target": "es5"',
+      '"target": "es5", "allowJs": true',
+    );
+
+    runTargetSpec(host, browserTargetSpec).pipe(
+      tap((buildEvent) => expect(buildEvent.success).toBe(true)),
+      tap(() => {
+        const content = virtualFs.fileBufferToString(
+          host.scopedSync().read(join(outputPath, 'main.js')),
+        );
+
+        expect(content).toContain(
+          `var x = __webpack_require__(/*! ./app.component.svg */ "./app.component.svg");`);
+        expect(content).toContain('module.exports = "<svg></svg>"');
+        expect(host.scopedSync().exists(normalize('dist/app.component.svg')))
+          .toBe(false, 'should not copy app.component.svg to dist');
+      }),
+    ).toPromise().then(done, done.fail);
+  });
+
+  it('imports non-component .svg with file-loader and copies .svg to dist', (done) => {
+
+    host.writeMultipleFiles({
+      './src/other.svg': `<svg></svg>`,
+      './src/main.ts': `declare var require: Function;
+      const x = require('./other.svg');
+      console.log(x);`,
+    });
+
+    host.replaceInFile(
+      'tsconfig.json',
+      '"target": "es5"',
+      '"target": "es5", "allowJs": true',
```

Comment:
Why allowJs is needed?

Reply:
That's just copied from the test that @filipesilva mentioned could be used as  example: https://github.com/angular/angular-cli/issues/10567#issuecomment-435126150

It's probably not necessary and can be removed.